Readme.md

```diff
@@ -0,0 +1,58 @@
+📖 Hello Microverse
+this application says hello to Microverse
+
+[Hello Microverse] is my first project in Microverse School 
+
+🛠 Built With
+--Html 
+--Css
+
+💻 Getting Started
+
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder.
+
+
+
+Usage
+To run the project, execute the index.html file.
+
+Run tests
+To run tests, run the following command:
+
+Deployment
+You can deploy this project using:
+
+
+
+👤 Author1
+
+GitHub: @fredo509
+Twitter: @Fredo_st_fleur
+LinkedIn: Fredo st fleur
+
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+⭐️ Show your support
+Please give me a star 
```

Comment:
- [ ] Your readme file is almost good but it doesn't look professional because of heading issues. Kindly fix the header of every section.

Your Readme             |  Readme Template
:-------------------------:|:-------------------------:
![readme](https://user-images.githubusercontent.com/70483835/218856477-fb0f1781-42a6-4888-ad6d-d0af9e38405e.png)  |  ![readme2](https://user-images.githubusercontent.com/70483835/218856528-725b3edc-ea81-4ff0-a52d-090fdc25413b.png)